### PR TITLE
Kb overlay customclass

### PR DESCRIPTION
**Pull request by bierik**
Make possible to add custom css class for overlay element.
Update german translations.

### CODE REVIEW

File: ftw/simplelayout/browser/resources/Overlay.js [hunk 1]
```diff
@@ -18,6 +18,7 @@
         left: "center",
         fixed: false,
         closeOnClick: false,
+        cssclass: "overlay-delete",
```
Comment (jone) [inline ftw/simplelayout/browser/resources/Overlay.js]: I dont think that `delete` is a good default - shouldn't this be defined where the delete overlay is invoked?

File: ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po [hunk 2]
```diff
@@ -82,6 +86,26 @@ msgstr "Video"
 msgid "Youtube format: http(s)://youtu.be/VIDEO_ID<br/>Vimeo format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
 msgstr "Youtube Format: http(s)://youtu.be/VIDEO_ID<br/>Vimeo Format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
 
+#. Default: "This item can not be deleted because it is currently locked by another user."
+#: ./ftw/simplelayout/browser/ajax/templates/block_delete_confirmation.pt:6
+msgid "alert_deleting_locked_item"
+msgstr "Dieser Eintrag kann nicht gelöscht werden da er gerade von einem anderen Benutzer blockiert ist."
```
Comment (jone) [inline ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po]: `blockiert` -> `gesperrt`.
Plone translates it with `gesperrt`, so we should be consistent with Plone translations.

File: ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po [hunk 3]
```diff
@@ -82,6 +86,26 @@ msgstr "Video"
...
+
+#. Default: "(This will delete a total of ${number_of_items_to_delete} items.)"
+#: ./ftw/simplelayout/browser/ajax/templates/block_delete_confirmation.pt:19
+msgid "alert_deleting_x_number_of_items"
+msgstr "(Diese Action wird ${number_of_items_to_delete} Einträge löschen.)"
```
Comment (jone) [inline ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po]: `Action` > `Aktion`

File: ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po [hunk 4]
```diff
@@ -110,18 +134,23 @@ msgstr "Typ"
...
 #: ./ftw/simplelayout/interfaces.py:82
 msgid "desc_sl_config_control_panel"
-msgstr ""
+msgstr "Simplelayout Standardkonfiguration hinzufügen, siehe Dokumentation: https://github.com/4teamwork/ftw.simplelayout#usage"
 
 #. Default: "sort_index"
 #: ./ftw/simplelayout/contents/filelistingblock.py:109
 msgid "label_%ssort_index"
-msgstr ""
+msgstr "Sortierungsindex"
```
Comment (jone) [inline ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po]: I'd go with `Sortierung` - index is too technical.

**Comment by bierik** [comment]
@jone thanks for the feedback

The css class is now a part of the options and be set individually for each overlay.

**Pull request merged**